lib/mixpanel-node.js

```diff
@@ -67,6 +68,10 @@ var create_client = function(token, config) {
 
         request_options.path = [endpoint,"?",query].join("");
 
+        //if the host and port have been changed, and the caller has also indicated this is a proxy
+        //then we want to pass through to the full http path
+        if (metrics.config.proxy) { request_options.path = "http://api.mixpanel.com"+request_options.path; }
```

Comment:
We should be able to get a patch in to do what you need, but this seems like an awfully specific interpretation of a param named `proxy`. Would you be OK with just passing a config option `path_prefix` instead? That way you can just pass the lib whatever host, port, and prefix you want.